code/modules/surgery/organs/cell_holder.dm

```diff
@@ -0,0 +1,136 @@
+/obj/item/organ/internal/cell_mount
+	name = "cell mount"
+	desc = "A small mechanical augment capable of holding a battery."
+	icon = 'icons/obj/robot_parts.dmi'
+	icon_state = "chest"
+	dead_icon = "chest"
+	organ_tag = "cell_mount"
+	parent_organ = "chest"
+	slot = "cell_mount"
+	status = ORGAN_ROBOT
+	var/obj/item/stock_parts/cell/cell
+
+/obj/item/organ/internal/cell_mount/full/New()
+	cell = new(src)
+	..()
+
+/obj/item/organ/internal/cell_mount/Destroy()
+	QDEL_NULL(cell)
+	return ..()
+
+/obj/item/organ/internal/cell_mount/attackby(obj/item/I, mob/user, params)
+	if(isscrewdriver(I) && cell)
+		to_chat(user, "<span class='notice'>You remove [cell] from [src].</span>")
+		cell.forceMove(get_turf(src))
+		cell = null
+		return
+	if(istype(I, /obj/item/stock_parts/cell))
+		if(!cell)
+			if(user.drop_item())
+				I.forceMove(src)
+				cell = I
+				to_chat(user, "<span class='notice'>You install [I] into [src].</span>")
+		else
+			to_chat(user, "<span class='notice'>[src] already has a cell.</span>")
+		return
+	return ..()
+
+/obj/item/organ/internal/cell_mount/on_life()
+	if(owner)
+		if(cell)
+			cell.use(1) // ~33 minutes to go from full to nothing
+			var/cellcharge = cell.charge/cell.maxcharge
+			switch(cellcharge)
+				if(0.75 to INFINITY)
+					owner.clear_alert("cell_organ")
+				if(0.5 to 0.75)
+					owner.throw_alert("cell_organ", /obj/screen/alert/lowcell, 1)
+				if(0.25 to 0.5)
+					owner.throw_alert("cell_organ", /obj/screen/alert/lowcell, 2)
+				if(0.01 to 0.25)
+					owner.throw_alert("cell_organ", /obj/screen/alert/lowcell, 3)
+				else
+					owner.throw_alert("cell_organ", /obj/screen/alert/emptycell)
+		else
+			owner.throw_alert("cell_organ", /obj/screen/alert/nocell)
+
+/obj/item/organ/internal/cell_mount/emp_act(severity)
+	if(emp_proof)
+		return
+	if(cell)
+		cell.emp_act(severity)
+
+/obj/item/organ/internal/cyberimp/arm/power_cord
```

Comment:
With charge now being a life-and-death matter for IPCs, alternative forms of recharging that aren't dependent on station infrastructure may be a good idea - like an alcohol-electricity converter, or some kind of plasma or uranium engine/reactor (although those two might not be good options for roundstart, since those materials are rare and contested).

Reply:
If we really want to go in depth, we could possibly make a few options like an ethanol engine, a cell syphon, or even a small engine like a Radioisotope Thermoelectric Generator (RTG) to either have as loadout options (either from a new option or the point system), printables, or give an item that lets them both pick and auto install. Maybe even give different systems benefits and downsides like the plasma heating up the air and releasing CO2, the ethanol engine letting out just a little CO2, and the RTG exiting radiation when on.